src/App.tsx

```diff
@@ -4,7 +4,7 @@ import { Dashboard, Login } from './pages';
 import { ProtectedRoute } from './core/components/ProtectedRoute';
 import { routes } from './core/routing/routes';
 import { NewStaffPage, StaffPage } from './staff/components';
-import { RootNavLayout } from './core/components/RootNavLayout';
+import { RootNavLayout } from './core/components/RootNavLayout/RootNavLayout';
```

Comment:
This import can be shortened as there is an index file in the RootNavLayout folder with the named re-exoport

Reply:
Import fixed.